### PR TITLE
fix ByteArray / felt252

### PR DESCRIPTION
**Pull request by TAdev0**
This change is [Reviewable](https://reviewable.io/reviews/cairo-book/cairo-book/762)

### CODE REVIEW

File: src/ch05-01-defining-and-instantiating-structs.md [hunk 1]
```diff
@@ -90,7 +90,7 @@ otherwise use the same values from `user1` that we created in Listing {{#ref use
 {{#label without-update-syntax}}
 
 <span class="caption">Listing {{#ref without-update-syntax}}: Creating a new `User` instance using all but one of
-the values from `user1`</span>
+the values from `user1`.</span>
```
Comment (TAdev0) [inline src/ch05-01-defining-and-instantiating-structs.md:93]: :p

Comment (enitrat) [reply src/ch05-01-defining-and-instantiating-structs.md:93]: that was on purpose, it's not a full sentence

Comment (enitrat) [reply src/ch05-01-defining-and-instantiating-structs.md]: that was on purpose, it's not a full sentence,

Comment (TAdev0) [reply src/ch05-01-defining-and-instantiating-structs.md]: just removed

Comment (enitrat) [reply src/ch05-01-defining-and-instantiating-structs.md]: https://en.m.wikipedia.org/wiki/Sentence_clause_structure#Incomplete_sentences

**Review by enitrat — APPROVED** [review]

**Pull request merged**